www/src/scss/navigator/shabad/_shabad.scss

```diff
@@ -38,3 +42,18 @@
     }
   }
 }
+
+.fav-btn {
+  background: $red;
+  height: 30px;
+  margin: 0 10px;
+  padding: 0px 10px 0 0;
```

Comment:
I know I'm knit picking 🙈 but 0 should be preferred instead of 0px

Reply:
Fixed